experimental/CHANGELOG.md

```diff
@@ -7,10 +7,16 @@ All notable changes to experimental packages in this project will be documented
 
 ### :boom: Breaking Change
 
+* fix(instrumentation-http): drop url.parse in favor of URL constructor [#5091](https://github.com/open-telemetry/opentelemetry-js/pull/5091) @pichlermarc
+  * (user-facing): signature of `getRequestInfo()` now requires a `DiagLogger` to be passed at the first position
```

Comment:
NOTE for reviewers: we're exporting lots of utils, I don't think this is intentional but it's still technically a breaking change.